presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowStatsRewrite.java

```diff
@@ -195,16 +212,16 @@ private Node rewriteShowStats(ShowStats node, Table table, Constraint<ColumnHand
             TableHandle tableHandle = getTableHandle(node, table.getName());
             TableStatistics tableStatistics = metadata.getTableStatistics(session, tableHandle, constraint);
             List<String> statisticsNames = findUniqueStatisticsNames(tableStatistics);
-            List<String> resultColumnNames = buildColumnsNames(statisticsNames);
-            List<SelectItem> selectItems = buildSelectItems(resultColumnNames);
-            Map<ColumnHandle, String> columnNames = getStatisticsColumnNames(tableStatistics, node, table.getName());
-
-            List<Expression> resultRows = buildStatisticsRows(tableStatistics, columnNames, statisticsNames);
+            List<String> statsColumnNames = buildStatsColumnsNames(statisticsNames);
+            List<SelectItem> selectItems = buildSelectItems(statsColumnNames);
+            Map<ColumnHandle, String> tableColumnNames = getStatisticsColumnNames(tableStatistics, tableHandle);
```

Comment:
renames `resultColumnNames -> statsColumnNames`, `columnNames -> tableColumnNames` could be a separate commit (but i don't insist)